test/fishjam_web/controllers/peer_controller_test.exs

```diff
@@ -80,9 +82,9 @@ defmodule FishjamWeb.PeerControllerTest do
     end
 
     test "renders errors when room doesn't exist", %{conn: conn} do
-      room_id = "invalid_room"
+      room_id = ID.generate()
       conn = post(conn, ~p"/room/#{room_id}/peer", type: @peer_type)
-      assert json_response(conn, :not_found)["errors"] == "Room #{room_id} does not exist"
+      assert json_response(conn, :not_found)["errors"] == "Room with this ID does not exist"
```

Comment:
I think it would be beneficial that we would return the passed ID, so the client could verify if it is a typo or what.

Reply:
Unfortunately, it's a PITA to achieve with the fallback controller...

Reply:
The room ID was extracted from params passed to the regular controller, and the fallback controller received a 3-tuple with the error string, where the room ID was already baked in. I don't know if it's possible to do that in the fallback controller without restoring the `else` cases that were removed for brevity and eliminating code duplication

Reply:
Refactored